### ISSUE TITLE
Bug: Improve mobile responsiveness for task management page

### ISSUE THREAD

**Issue by Ninjaabhay** [Ninjaabhay/projectSheManages#2]
- **Description**: The task management page does not display properly on smaller screens like mobile devices. Buttons and text fields are overlapping or misaligned, making it difficult to interact with tasks.
   - **Steps to Reproduce**:  
     1. Open the app on a mobile device.  
     2. Navigate to the task management page.  
     3. Observe the alignment issues with text and buttons.
   - **Expected Behavior**: The page should be fully responsive, with elements adapting to the screen size without overlap.
   - **Solution**: Implement responsive design techniques using CSS media queries or frameworks like Tailwind CSS to ensure proper display on different screen sizes.

---

**Comment by yash25hemnani** [comment]
Hey, can you assign me this issue as part of Hacktoberfest? I will commit the solution in the first few days of October.

**Comment by Ninjaabhay** [comment]
Hi @yash25hemnani assigned the issue, looking forward to your meaning full contribution.

**Comment by yash25hemnani** [comment]
Thank you, will get right to it.

**Comment by yash25hemnani** [comment]
Hey @Ninjaabhay, which one is the task management page?

**Comment by yash25hemnani** [comment]
Also, I want to suggest a color scheme for the project, I can implement that as well if you prefer, otherwise I will go along with the current theme.
Here is the link to the theme - https://www.instagram.com/p/DFcIXXYSFBi/

**Comment by Ninjaabhay** [comment]
@yash25hemnani . Sorry I forgot to close that issue. We'd deprecated that task management page idea actually so we don't have that page right now.

**Comment by Ninjaabhay** [comment]
But we can implement one if anyone wants to contribute.

**Comment by Ninjaabhay** [comment]
@yash25hemnani yes we can change the theme but it shouldn't be completely opposite to the current theme, we can use a color palette which is much familiar with the current one.

**Comment by yash25hemnani** [comment]
Okay, understood.

For the Task Management, you can raise a new feature issue. That would be great.

**Comment by Ninjaabhay** [comment]
Created #8

**Comment by yash25hemnani** [comment]
Hey @Ninjaabhay, here are some changes I made, are you okay with them or should I revert back - 

<img width="1783" height="883" alt="Image" src="https://github.com/user-attachments/assets/7adee363-7aca-4401-be8d-62063f53c5ea" />

<img width="1783" height="883" alt="Image" src="https://github.com/user-attachments/assets/bb2235b9-02a3-4db4-884f-887ef4325d36" />

<img width="1783" height="883" alt="Image" src="https://github.com/user-attachments/assets/2b4b0e14-fa4e-4a76-9bbd-1c13b5fb68a7" />

<img width="1783" height="883" alt="Image" src="https://github.com/user-attachments/assets/9828ce92-9e73-4792-8faf-9454c0280052" />

**Comment by Ninjaabhay** [comment]
@yash25hemnani ahh I see you made some changes in font color that looks good. but rather focusing on minor tweeks we can focus more towards the functionality.

**Comment by Ninjaabhay** [comment]
you can continue this tweeked theme it looks good.

**Comment by yash25hemnani** [comment]
Thanks for the input, I've already implemented Clerk auth, working on its corresponding backend currently.

**Comment by yash25hemnani** [comment]
Also, by default clerk supports email password and full name, do you want me to make a custom form or more preferably let the user enter the bio, phone and image in the profile page directly. This would reduce the overall development time and make the flow more streamlined.

**Comment by Ninjaabhay** [comment]
> Thanks for the input, I've already implemented Clerk auth, working on its corresponding backend currently.

Thats great.

**Comment by Ninjaabhay** [comment]
> Also, by default clerk supports email password and full name, do you want me to make a custom form or more preferably let the user enter the bio, phone and image in the profile page directly. This would reduce the overall development time and make the flow more streamlined.

Yeah sounds good we can input more info into the profile page directly after the auth we should redirect the user to the onboarding/profile page where they can input further details.

**Comment by yash25hemnani** [comment]
Okay done.
